### ISSUE TITLE
multiple tables

### ISSUE THREAD

**Issue by Muhahe** [vedmack/yadcf#205]
Hello again. Amazing work on multiple tables, but im still geting some strange error until i made some changes, maybe you should know what to do, but my brain getting hard shocks from it :p

When i initialize datatable like this

```javascript
   var subTable = $(subtableElement).dataTable({ etc...
```

then im storing variables subTable into array and in another part of code im initializing yadcf like this

```javascript
function initializeMultifilterForSubtables(){

    yadcf.initMultipleTables(subTablesList,[{column_number: [1], filter_container_id:'multiFilterContainer_1', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [2], filter_container_id:'multiFilterContainer_2', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [3], filter_container_id:'multiFilterContainer_3', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [4], filter_container_id:'multiFilterContainer_4', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [5], filter_container_id:'multiFilterContainer_5', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [6], filter_container_id:'multiFilterContainer_6', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [7], filter_container_id:'multiFilterContainer_7', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [8], filter_container_id:'multiFilterContainer_8', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [9], filter_container_id:'multiFilterContainer_9', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                                            {column_number: [11], filter_container_id:'multiFilterContainer_11', filter_type: 'multi_select', column_data_type: 'rendered_html', select_type: 'chosen',filter_reset_button_text: false,filter_default_label: "Select"},
                        ])
}

```

with this im getting error Uncaught TypeError: tablesArray[i].table is not a function on row 3226

if i fix it with

```javascript
                            tablesSelectors += '#' + tablesArray[i][0].id + ',';
				//tablesSelectors += tablesArray[i].table().node().id + ',';
```

im getting Uncaught TypeError: tablesArray[tableTmpArrIndex].table is not a function on 3105

and fix like this
```javascript
                       tableTmp = tablesArray[tableTmpArrIndex]
					//tableTmp = $('#' + tablesArray[tableTmpArrIndex].table().node().id).dataTable();

```

then yadcf will initialize everything seems ok, but yadcf wont recovery savedState. Datatables remember it, but multipleSelectBoxes are empty

All this tables are subtables nested in one big table (some of them are out of dom) and big table works fine. But its initialized with capital D (.DataTables())

**Comment by vedmack** [comment]
Hi,

Can it be that you are using an old datatables version? because you should use 1.10

**Comment by Muhahe** [comment]
If im understanding everything correctly, im using datatables 1.10.4, but only different constructor with non-capital D. Should it be problem even in 1.10.x?

**Comment by vedmack** [comment]
fixed in [0.8.9.beta.9](https://github.com/vedmack/yadcf/blob/a36df45bec3d8490c93746d23ea64dcfb6786a66/jquery.dataTables.yadcf.js) took me a while I know :)
